packages/gestalt/src/IconButton.js

```diff
@@ -87,9 +107,75 @@ const IconButtonWithForwardRef: React$AbstractComponent<
   const classes = classnames(styles.button, touchableStyles.tapTransition, {
     [styles.disabled]: disabled,
     [styles.enabled]: !disabled,
-    [touchableStyles.tapCompress]: !disabled && isTapping,
+    [touchableStyles.tapCompress]:
+      props.role !== 'link' && !disabled && isTapping,
   });
 
+  function handleClick(event) {
+    if (onClick) {
+      onClick({ event });
+    }
+  }
+
+  function handleLinkClick({ event }) {
+    handleClick(event);
+  }
+
+  const renderPogComponent = (selected?: boolean): Node => {
```

Comment:
@AlbertCarreras this looks like a React component but it's not. Instead can we make it a real component?

`const PogWithSelected = ({props}): Node =>`

`<PogWithSelected selected="..." />`

Reply:
DONE